data/json/bionics.json

```diff
@@ -750,15 +750,15 @@
     "name": "Power Storage",
     "capacity": 100,
     "description": "Compact Bionics Module that upgrades your power capacity by 100 units.  Having at least one of these is a prerequisite to using powered bionics.  You will also need a power supply, found in another CBM.",
-    "flags": [ "BIONIC_NPC_USABLE" ]
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_FIRE_AND_FORGET" ]
```

Comment:
Please add documentation for this flag.

Reply:
Huh, I thought I did. Yeah, I'll do that when I change the flag name like Pryanik asked.

I generally try to leave a mark of `// Aftershock` when defining things for the mod itself, but I could try to implement it for each particular occurrence - I just worry that it might get spammy.

Reply:
In my opinion, spammy but clear is better than clean, but unclear.